### ISSUE TITLE
Repeat result with attention mask warning

### ISSUE THREAD

**Issue by shenxiangzhuang** [FunAudioLLM/Fun-ASR#10]
The warning message:

```
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
````

The output:

```
嗯，对，就是那个功能，就是那个功能，就是那个功能，就是那个功能，就是那个功能, ...
```

**Comment by huanchen-1** [comment]
on my testing, using VAD can avoid this problem

**Comment by ljxqxq** [comment]
`  0%|          | 0/1 [00:00<?, ?it/s]The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.`

the same warning message

**Comment by LauraGPT** [comment]
could you upload the wav file?

**Comment by xztzmr** [comment]
> on my testing, using VAD can avoid this problem

Could you please tell me how you use the vad model? Thank you very much.
